MenuTest/Menu/MenuView.swift

```diff
@@ -264,10 +264,10 @@ public class MenuView: UIView, MenuThemeable {
     
     private func relayoutContents() {
         if let contents = contents {
-            self.setNeedsLayout()
-            self.layoutIfNeeded()
+            setNeedsLayout()
```

Comment:
Just a note that removing `self` here doesn't do anything to relate to memory leaks or strongly capturing self. It's simply a matter of style.

But, having said that, I much prefer writing without `self` where possible so this is a good change.